client/src/pages/Find.jsx

```diff
@@ -1,25 +1,67 @@
 import { useEffect, useState } from 'react';
 import Header from '../components/Header';
 import Navigation from '../components/Navigation';
+import Filter from '../components/Filter';
 import BoulderCard from '../components/BoulderCard';
 import styled from 'styled-components';
 const URL = process.env.REACT_APP_URL;
 
-const Find = () => {
+const BoulderList = styled.ul`
```

Comment:
I recommend to put Styled Components to the end of the document.

Reply:
Changed